packages/shared/src/components/Settings.tsx

```diff
@@ -64,7 +62,7 @@ export default function Settings({
   ...props
 }: HTMLAttributes<HTMLDivElement>): ReactElement {
   const isExtension = checkIsExtension();
-  const isFeedV1 = useFeature(feature.feedLayout) === FeedLayout.V1;
+  const { shouldUseFeedLayoutV1 } = useFeedLayout({ feedRelated: false });
```

Comment:
> Other idea is to wrap the component in ActiveFeedNameContext.Provider and set some other field name value there.

This doesn't really makes sense as it's not feed related. We want to disable this in general for all feeds while we have the experiment running. 

> Just using useFeature directly feels best to me.

My issue with this is not having one place to configure and make checks for a certain experiment. I've been struggling with enabling and disabling search locally when checking if the design is ok, since the checks for the feature versions are in different places.
Or there is an easier way to say locally that we are on a certain experiment ( search V1?)

Reply:
> Or there is an easier way to say locally that we are on a certain experiment ( search V1?)

`useFeature` should do that?

Reply:
Yeah just modifying the default featureManagement to whatever you need should do just that.
That way wherever you use useFeature adopts automatically.

Reply:
🤦‍♀️